proposal.rst

```diff
@@ -200,27 +195,27 @@ Project Details
 
 **Describe any existing work and concepts on which your project is based.**
 
-FILLME
+	I have studied with Golang in my internship. I worked on developing a software that makes web crawling.	I used PostgreSQL and Beego in this project.
+
 
 **Do you have any preliminary findings or results which suggest that your
 approach is possible and likely to succeed?**
 
-FILLME
+-
 
 **What other approaches have you considered, and why did you reject those in
 favour of your chosen approach?**
 
-FILLME
+	I thought of finding only the best fit for the approaches.
 
-**Please note any uncertainties or aspects which depend on further research or
-investigation.**
+**Please note any uncertainties or aspects which depend on further research or investigation.**
 
 FILLME
 
 **How useful will your results be when not everything works out exactly as
 planned?**
 
-FILLME
+	I think my results will be useful because I don't lose motivation in a stress situation.
 
 Project Timeline
 ----------------
```

Comment:
You've not filled in a timeline for your project at all.

The timeline is a key part of the proposal, as it shows what you're planning to do, and that you've thought about how much you can achieve in the time available.  It also provides a way for us and for you to see how on track the project is as it progresses, which is important for motivation, and for knowing if we need to re-plan if things take longer than expected (or to know that you'll need to find more to do if things take less time than expected).